requirements.txt

```diff
@@ -3,3 +3,4 @@
 # pip install -r requirements.txt
 #---------------------------------------------------------
 requests
+websocket-client>=1.3.3
```

Comment:
`websocker-client` was removed from `requirements.txt` in #468 (and discussion in #428).